spec/client_spec.rb

```diff
@@ -209,7 +209,7 @@
       end
 
       context 'with non-serializable data' do
-        let(:data) { [:foo, 'bar'] }
```

Comment:
Principle of least magic, I guess — if someone shoves a symbol in, they could expect to get a symbol out (and they wouldn't).